### ISSUE TITLE
Input of web access password seems to fail

### ISSUE THREAD

**Issue by musevg** [dtag-dev-sec/t-pot-autoinstall#26]
Hi there,

I tried to use the installer do get T-Pot on a Ubuntu 16.04.2 at Kimsufi/OVH.
I installed option 4, and the script failed after unpacking the packages when I entered the 
web access password "Web Access Password":
```
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
### Removing NGINX default website.
### Please enter a password for your user spx for web access.
Password:
Repeat password:
Usage:
        htpasswd [-cimBdpsDv] [-C cost] passwordfile username
        htpasswd -b[cmBdpsDv] [-C cost] passwordfile username password

        htpasswd -n[imBdps] [-C cost] username
        htpasswd -nb[mBdps] [-C cost] username password
 -c  Create a new file.
 -n  Don't update file; display results on stdout.
 -b  Use the password from the command line rather than prompting for it.
 -i  Read password from stdin without verification (for script usage).
 -m  Force MD5 encryption of the password (default).
 -B  Force bcrypt encryption of the password (very secure).
 -C  Set the computing time used for the bcrypt algorithm
     (higher is more secure but slower, default: 5, valid: 4 to 31).
 -d  Force CRYPT encryption of the password (8 chars max, insecure).
 -s  Force SHA encryption of the password (insecure).
 -p  Do not encrypt the password (plaintext, insecure).
 -D  Delete the specified user.
 -v  Verify password for the specified user.
On other systems than Windows and NetWare the '-p' flag will probably not work.
The SHA algorithm does not use a salt and is less secure than the MD5 algorithm.
```
This behaviour is reproducible (at least here), and it seems to be caused by the blanks in 
the quoted password. 
So you may want to add a warning, not to use blanks in the password.

Thanks,
M

**Comment by vorband** [comment]
as you are the first to notice, it's probably not an issue for others.

**Comment by musevg** [comment]
Apparently most other users just don't use blanks in their web passwords - I just did once more and the installer failed once more. 
I still recommend adding at least a hint like `Don't use blanks in your web password or install.sh will fail` to the install script and/or the documentation.

**Comment by t3chn0m4g3** [comment]
@vorband After the Ansible change the password / user vars are not in hyphens, hence everything after the space is ignored or leads to errors.

**Comment by vorband** [comment]
sorry, fixed in 2bd047b84122c8703de357f2f754ca5473824175 
thanks.
